adatest/_test_tree_browser.py

```diff
@@ -68,11 +68,11 @@ def file_log(*args):
     f.flush()
     f.close()
 
-def is_subtopic(topic, candidate):
+def is_subtopic(topic: str, candidate: str) -> bool:
```

Comment:
I'm seeing what looks like a copy of this in `_topic_model.py` ?

Reply:
Good catch on the replicated function in `_topic_model.py` -- I believe that's dead code, so this is the one we should keep.

Reply:
There's another in `_topic_model.py`. I'll leave alone for now, but that's something we should tidy up.